### DISCUSSION TITLE
Support for custom formats as defined in the specification

### DISCUSSION THREAD

**Discussion by gayanper** [OAI/OpenAPI-Specification#2730] [Q&A] [answered]
The specification says about format as follows
`format – OpenAPI has its own predefined formats and also allows custom formats.`

But how exactly a custom format can be used? I don't find any features in the tooling of OpenAPI to extend support for custom formats that we use in a spec.

**Comment by webron** [comment, marked as answer]
That's more of a tooling question than a spec one. Glad to hear you have usage for such a feature though. My recommendation is to reach out to the maintainers of the tools you use and ask them to support it.

As a side note - we recently created a [Special Interest Group for Formats](https://github.com/OAI/sig-formats) where we'll be discussing how to create some sort of a format registry to help OpenAPI users.

**Comment by gayanper** [comment]
Request is created at tooling https://github.com/OpenAPITools/openapi-generator/issues/10539

**Comment by handrews** [comment]
Resolved as far as  we can go, plus issue opened with tooling vendor.  Closing.
